eos-tech-support/eos-diagnostics

```diff
@@ -556,7 +556,7 @@ let diagnostics = [
     {
         title: 'Journal',
         hardwareInfo: true,
-        content: function() { return trySpawn('journalctl --no-pager'); },
+        content: function() { return trySpawn('journalctl --no-pager --output verbose'); },
```

Comment:
Made it optional behind a `--verbose` flag (but it’s also enabled if running with `--coredumps`).